routes.py

```diff
@@ -57,48 +72,84 @@ def mine():
     db.Put(('block-' + str(block['index'])).encode(), json.dumps(block, sort_keys=True).encode())
     print(db.Get(('block-' + str(block['index'])).encode()).decode())
 
-    # response = {
-    #     'message': "New Block Forged",
-    #     'index': block['index'],
-    #     'transactions': block['transactions'],
-    #     'proof': block['proof'],
-    #     'previous_hash': block['previous_hash'],
-    # }
-    # return jsonify(response), 200
+    # return render_template('wallet.html')
+
+
+@app.route('/')
+def home():
+    return render_template('login.html')
+
+
+@app.route('/wallet')
+def wallet():
+    return render_template('wallet.html')
+
+
+@app.route('/login', methods=['POST'])
+def login():
+    global privkey, pubkey, wallet
+    keyword = request.form['keyword']
+
+    # Create privatekey, publickey
+    keywordByte = binascii.hexlify(keyword.encode())
```

Comment:
Please use [snake case](https://en.wikipedia.org/wiki/Snake_case). It is a common python style convention.